src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract String doWork();
+
+    public abstract String stopWork();

```

Comment:
These methods should not return a string
```suggestion
    public abstract void doWork();

    public abstract void stopWork();
```
<img width="848" alt="Знімок екрана 2023-03-07 о 20 42 27" src="https://user-images.githubusercontent.com/41166578/223519346-b871d82c-0c21-4eba-8ca7-fd1af51006b4.png">